bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -26,6 +26,9 @@ class CategoricalAxis extends Axis.Model
 
     return range_bounds
 
+  display_defaults: () ->
+    return super()
+
```

Comment:
I see it now. It's very subtle and I'm sure it will cause trouble next person touching display properties. I think we can't leave it like this.